Rise-Vision/rise-video
# Feature/preview mode image

## Description

Show a static icon instead of playing videos when in preview mode.

## Motivation and Context

Work related to [this Trello card](https://trello.com/c/f7a0dDL1).

We need to show a static icon when in preview mode in the template editor. I spoke with Jason regarding the icon to use and its size / color.

## How Has This Been Tested?

Unit tests have been added to cover the changes, and the change has been manually tested on Staging in these browsers:

- Chrome 76
- Firefox 68
- Safari 12.1.2
- Mobile Safari 12.1.2 on iPhone X
- Mobile Chrome 76 on Pixel 2

Changes have been pushed to GCS [here](https://widgets.risevision.com/staging/components/rise-video/2019.08.07.08.07/rise-video.js) and a test template which uses these changes is available [here](https://apps.risevision.com/templates/edit/6f325009-7d10-44fa-9e68-bb90301cac73/?cid=7fa5ee92-7deb-450b-a8d5-e5ed648c575f).

## Release Plan:

- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
  - Manual Test: Completed as noted above
  - Automated Test: None, as noted above
  - Monitoring: I will validate all changes in Production after deployment and check the BQ logs for a few days after release
  - Rollback: Revert to the previous commit version (v1.0.7)
  - Documentation: No documentation updates required
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?

No release checklist items were skipped.
